Command/MigrateCommand.php

```diff
@@ -120,57 +98,35 @@ protected function configure()
                 InputOption::VALUE_OPTIONAL,
                 'Query used to find elasticsearch records to import',
                 '{"sort":{"_uid":{"order":"asc"}}}'
+            )
+            ->addOption(
+                'dont-finalize',
+                null,
+                InputOption::VALUE_NONE,
+                'Don\'t finalize document'
             );
-        ;
-    }
-    
-    
-    private function getSubmitData(FormInterface $form)
-    {
-        return $this->dataService->getSubmitData($form);
-    }
-    
-    private function getEmptyRevision(ContentType $contentType)
-    {
-        return $this->dataService->getEmptyRevision($contentType, 'SYSTEM_MIGRATE');
     }
 
-    /**
-     * @param InputInterface $input
-     * @param OutputInterface $output
-     * @return int|void|null
-     * @throws MappingException
-     * @throws ORMException
-     * @throws OptimisticLockException
-     */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        /** @var EntityManager $em */
-        $em = $this->doctrine->getManager();
-        //https://stackoverflow.com/questions/9699185/memory-leaks-symfony2-doctrine2-exceed-memory-limit
-        //https://stackoverflow.com/questions/13093689/is-there-a-way-to-free-the-memory-allotted-to-a-symfony-2-form-object
-        $em->getConnection()->getConfiguration()->setSQLLogger(null);
-        
-        
-        $signData = $input->getOption('sign-data');
-        
         $elasticsearchIndex = $input->getArgument('elasticsearchIndex');
```

Comment:
would be easier on the eyes using the lists($...) = $input->getArguments(); trick.

Reply:
5e9c4b0e1954bc45fa2715255d50a6080da6c862